sdk/storage/azure-storage-blob/azure/storage/blob/_version.py

```diff
@@ -4,4 +4,4 @@
 # license information.
 # --------------------------------------------------------------------------
 
-VERSION = "12.26.0b1"
+VERSION = "12.26.0"
```

Comment:
Let's do `12.25.1` instead since it's a minor change. Don't forget to update datalake dependency.